### PR TITLE
DOCS-4925 Software prereq info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by hannanelson — APPROVED** [review]
LGTM (except "may")

**Review by federicoboerr — APPROVED** [review]
LGTM

**Comment by paigehargrave** [comment]
Thank you @hannanelson and @federicoboerr !

File: modules/ROOT/pages/prereq-verify-disk.adoc [hunk 1]
```diff
@@ -54,7 +54,7 @@ The `dd` command provides less accurate information than `hdparm`, but is availa
 
 == Testing Disk IOPS
 
-Depending on your hardware or virtualization provider, you may be able to configure disk IOPS (I/O Operations Per Second). Using the `iops` command, you can verify available IOPS:
+Depending on your hardware or virtualization provider, you can configure disk IOPS (I/O Operations Per Second). Using the `iops` command, you can verify available IOPS:
```
Comment (dhirsch1) [inline modules/ROOT/pages/prereq-verify-disk.adoc]: It might be better to say "Use the `logs` command to verify....

**Pull request merged**